lte/gateway/deploy/agw_install_ubuntu.sh

```diff
@@ -0,0 +1,170 @@
+#!/bin/bash
+# Copyright 2021 The Magma Authors.
+
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Setting up env variable, user and project path
+MAGMA_USER="magma"
+AGW_INSTALL_CONFIG="/etc/systemd/system/multi-user.target.wants/agw_installation.service"
+AGW_SCRIPT_PATH="/root/agw_install_ubuntu.sh"
+DEPLOY_PATH="/home/$MAGMA_USER/magma/lte/gateway/deploy"
+SUCCESS_MESSAGE="ok"
+NEED_REBOOT=0
+WHOAMI=$(whoami)
+MAGMA_VERSION="${MAGMA_VERSION:-v1.5}"
+CLOUD_INSTALL="cloud"
+GIT_URL="${GIT_URL:-https://github.com/magma/magma.git}"
+
+
+echo "Checking if the script has been executed by root user"
+if [ "$WHOAMI" != "root" ]; then
+  echo "You're executing the script as $WHOAMI instead of root.. exiting"
+  exit 1
+fi
+
+wget https://raw.githubusercontent.com/magma/magma/"$MAGMA_VERSION"/lte/gateway/deploy/agw_pre_check_ubuntu.sh
```

Comment:
we need to implement something like https://github.com/magma/magma/pull/6139 otherwise the script will be stuck in a loop after reboot